packages/webcomponents/src/components/business-form/business-form-schema.ts

```diff
@@ -111,7 +111,7 @@ export const BusinessStructureOptions: {
     },
   ];
 
-export const businessGenericInfoSchema = object({
```

Comment:
Can we start breaking this up into files that can go alongside the appropriate components? And anything that is shared can stay here, though maybe it will make sense to change the file name at that point

Reply:
I was planning on doing another cleanup PR that would address things like this file, and making sure that we have items that are shared between `business-form` and `business-form-stepped` in their appropriate place. 

We can totally do it in this file if you prefer, but I wanted to keep this PR mostly focused on the form sub-components managing their own Form and data